features/hq/hq_features/authentication.feature

```diff
@@ -0,0 +1,14 @@
+Feature:
+  As a site admin for osra
+  So that I have access to restricted pages
+  I would have to authenticate first
```

Comment:
Since the key intent there is to show that protected pages _are not_ accessible without authentication, it may be more revealing to re-write this as something like
```
As an OSRA admin
So that I can be sure that the application is not accessible to the public
I would like to see that protected pages cannot be views without authentication
```